draft/draft-ietf-ippm-twamp-yang.xml

```diff
@@ -821,7 +821,7 @@ module: ietf-twamp
                 dynamic-port-number
              +--ro reflector-ip                         inet:ip-address
              +--ro reflector-udp-port \
-                dynamic-port-number
+                uint32

```

Comment:
Again, no need to make the change in the tree. The run of 'make' generates the tree from the model.